app/src/main/res/layout/activity_album_review.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              xmlns:tools="http://schemas.android.com/tools"
+              android:orientation="vertical"
+              android:weightSum="1"
+              android:background="@drawable/album_border"
+              android:padding="@dimen/album_art_border">
+
+    <LinearLayout
```

Comment:
Why do you need this `LinearLayout` here? It has the same orientation as it's parent so I feel like this could be replaced with something simpler like a `FrameLayout` in case you only use it for the border